vpr/src/place/move_transactions.cpp

```diff
@@ -126,10 +137,10 @@ void clear_move_blocks(t_pl_blocks_to_be_moved& blocks_affected) {
     blocks_affected.moved_to.clear();
     blocks_affected.moved_from.clear();
 
-    //For run-time, we just reset num_moved_blocks to zero, but do not free the blocks_affected
+    //For run-time, we just reset size of blocks_affected.moved_blocks to zero, but do not free the blocks_affected
     //array to avoid memory allocation
 
-    blocks_affected.num_moved_blocks = 0;
+    blocks_affected.moved_blocks.resize(0);
```

Comment:
clear_move_blocks() can also be implemented as a method of t_pl_blocks_to_be_moved.

Reply:
if clear_move_blocks() and clear_move_blocks() are implemented as member functions of t_pl_blocks_to_be_moved, we can make all its member variable private.